samples/http-client-examples/server.ts

```diff
@@ -1,14 +1,5 @@
-import express from "express";
+const express = require("express");
```

Comment:
[q] require in typescript? Even if they do not have a named export you can use `import * as express from 'express'` which should be the same.

Reply:
I think I had some issues with esm/cjs related things and used require because of that, but using import seems to work just fine as is.. updated, thanks!